SSA/Projects/InstCombine/splitandprocess.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+def getProofs(lines):
+    proofs = []
+    
+    proof = []
+    for line in lines:
+        if line.startswith("-- Name"):
+            proofs.append(proof)
+            proof = []
+        proof.append(line)
+    proofs.append(proof)
+
+    return proofs[0], proofs[1:]
+
+def isWorkingProof(preamble, proof):
+    f = open("InstCombineAliveTest.lean", "w")
+
+    f.write("".join(preamble))
+    f.write("".join(proof))
+
+    f.close()
+    
+    import subprocess
+    x = subprocess.run("(cd ../../../; lake build SSA.Projects.InstCombine.InstCombineAliveTest)", shell=True)
```

Comment:
If we want to be fancy, we can parallelize this by writing the different tests to different files `test_n.lean` and then compiling each of the tests in parallel with `lake env lean /path/to/test_n.lean`. This will compile the test file (using `lean`) in the project environment  (`lake env`).